CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
@umerkhan-apple you can reproduce using this [branch](https://github.com/matteocrippa/CareKit/tree/tintColorIssue) , as you can see the only component that goes purple is the name of the day, nor Today button or other tint Object are purple as expected.

![simulator screen shot 4 may 2016 20 12 55](https://cloud.githubusercontent.com/assets/475463/15024358/d19aa830-1234-11e6-91e0-51dc6fda8314.png)